### PR TITLE
testing

### PR DESCRIPTION
**Pull request by wata727**

### CODE REVIEW

File: app/controllers/foo_controller.rb [hunk 1]
```diff
@@ -3,6 +3,7 @@ def new_user
     user = User.first(:conditions => ("username = '#{params[:username]}'"))
     user = User.first(:conditions => ("username = '#{params[:username]}'"))
     user = User.first(:conditions => ("username = '#{params[:username]}'"))
+    user = User.first(:conditions => ("username = '#{params[:username]}'"))
```
Comment (sumyapp) [inline app/controllers/foo_controller.rb:6]: **[brakeman]**
- Possible SQL injection [:ref](http://brakemanscanner.org/docs/warning_types/sql_injection/)

Comment (sumyapp) [inline app/controllers/foo_controller.rb:6]: **[reek]**
- [DuplicateMethodCall] calls User.first(:conditions => ("username = '#{params[:username]}'")) 4 times [:ref](https://github.com/troessner/reek/blob/master/docs/Duplicate-Method-Call.md)
- [DuplicateMethodCall] calls params[:username] 4 times [:ref](https://github.com/troessner/reek/blob/master/docs/Duplicate-Method-Call.md)

Comment (wata727) [inline app/controllers/foo_controller.rb:6]: **[staging][rubocop]**
- Use the new Ruby 1.9 hash syntax. [:ref](https://github.com/bbatsov/ruby-style-guide#hash-literals)

**Comment by wata727** [comment]
Click here if you want to create rubocop's auto-correct Pull Request 
[staging] 
[Auto Correct by SideCI](https://www.codemagica.com/gh/38860676/issues/pull_request/21/diff/45ed9a3252869d5948e7b3c98665662a6874c4c9?open_modal=1#rubocop)

See the documentation if you want to get more information about auto-correct.
[Document](https://www.codemagica.com/ja/docs/autofix-rubocop)

File: app/controllers/foo_controller.rb (hunk 1 above)
Comment (sumyapp) [inline app/controllers/foo_controller.rb:6]: **[rubocop]**
- Use the new Ruby 1.9 hash syntax. [:ref](https://github.com/bbatsov/ruby-style-guide#hash-literals)

Comment (wata727) [inline app/controllers/foo_controller.rb:6]: **[staging][brakeman]**
- Possible SQL injection [:ref](http://brakemanscanner.org/docs/warning_types/sql_injection/)

**Comment by sumyapp** [comment]
Click here if you want to create rubocop's auto-correct Pull Request 
[Auto Correct by SideCI](https://sideci.com/gh/38860676/issues/pull_request/21/diff/45ed9a3252869d5948e7b3c98665662a6874c4c9?open_modal=1#rubocop)

See the documentation if you want to get more information about auto-correct.
[Document](https://sideci.com/ja/docs/autofix-rubocop)

**Comment by wata727** [comment]
Created Auto-correct Pull Request from [SideCI](https://sideci.com/)

[https://github.com/actcat/sideci_rails_sandbox/pull/22](https://github.com/actcat/sideci_rails_sandbox/pull/22)

**Comment by wata727** [comment]
Auto-correctable file didn't exist on this Pull Request from [SideCI](https://www.sideci.com/).

**Pull request closed without merging**